src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/eventIdField.tsx

```diff
@@ -8,57 +8,76 @@ import space from 'app/styles/space';
 import Field from 'app/views/settings/components/forms/field';
 
 import EventIdFieldStatusIcon from './eventIdFieldStatusIcon';
-import {EventIdStatus, EventId} from '../types';
+import {EventIdStatus, EventId} from '../../types';
+import {saveToSourceGroupData} from '../utils';
 
 type Props = {
   onUpdateEventId: (eventId: string) => void;
-  eventId?: EventId;
+  eventId: EventId;
   disabled?: boolean;
 };
 
 type State = {
   value: string;
-  status?: EventIdStatus;
+  status: EventIdStatus;
 };
 
 class EventIdField extends React.Component<Props, State> {
-  state = {
-    value: this.props.eventId?.value || '',
-    status: this.props.eventId?.status,
-  };
+  state: State = {...this.props.eventId};
 
   componentDidUpdate(prevProps: Props) {
     if (!isEqual(prevProps.eventId, this.props.eventId)) {
       this.loadState();
     }
   }
 
-  loadState = () => {
+  loadState() {
     this.setState({
       ...this.props.eventId,
     });
-  };
+  }
 
-  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const eventId = event.target.value.replace(/-/g, '').trim();
+  getErrorMessage(): string | undefined {
+    const {status} = this.state;
 
-    if (eventId !== this.state.value) {
-      this.setState({
-        value: eventId,
-        status: undefined,
-      });
+    switch (status) {
+      case EventIdStatus.INVALID:
+        return t('This event ID is invalid.');
+      case EventIdStatus.ERROR:
+        return t(
+          'An error occurred while fetching the suggestions based on this Event ID.'
```

Comment:
```suggestion
          'An error occurred while fetching the suggestions based on this event ID.'
```

Event is not upper case in the other error messages.